main.tf

```diff
@@ -4,6 +4,7 @@ provider "aws" {
 }
 
 resource "aws_iam_user" "user" {
```

Comment:
Please rename the `"user"`  `"this"` to keep that output paradigm with `this_user_arn` etc. tip top.

Reply:
nice catch! done.